src/mgmt/srv_pool.c

```diff
@@ -391,14 +392,15 @@ ds_mgmt_pool_extend(uuid_t pool_uuid, d_rank_list_t *svc_ranks,
 
 int
 ds_mgmt_evict_pool(uuid_t pool_uuid, d_rank_list_t *svc_ranks,
-		   const char *group)
+		   uuid_t *handles, size_t n_handles, const char *group)
 {
 	int		 rc;
 
 	D_DEBUG(DB_MGMT, "evict pool "DF_UUID"\n", DP_UUID(pool_uuid));
 
 	/* Evict active pool connections if they exist*/
-	rc = ds_pool_svc_check_evict(pool_uuid, svc_ranks, true);
+	rc = ds_pool_svc_check_evict(pool_uuid, svc_ranks, handles, n_handles,
```

Comment:
this is a fairly new use of ds_pool_svc_check_evict(), beyond its original use only in pool destroy execution flow. Here, I think it should have been called with force=false (if only for clarity for a code reader). Can that change be made in this patch?
i.e., the force argument is only applicable if destroy=true, so why not specify force=false here (functionally it doesn't matter that it is passed force=true).

Reply:
Ill make that change. It was probably just a copy paste error.